components/core/src/os/process/windows.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::ffi::OsString;
+use std::path::PathBuf;
+use std::process::{self, Command};
+
+use error::Result;
+
+pub fn become_command(command: PathBuf, args: Vec<OsString>) -> Result<()> {
+    become_child_command(command, args)
+}
+
+/// Executes a command as a child process and exits with the child's exit code.
+///
+/// Note that if sucessful, this function will not return.
+///
+/// # Failures
+fn become_child_command(command: PathBuf, args: Vec<OsString>) -> Result<()> {
+    debug!("Calling child process: ({:?}) {:?}",
+           command.display(),
+           &args);
+    let mut command = Command::new(command);
+    for arg in &args {
```

Comment:
No need to iterate over and call `arg()` for each, there's [args()](https://doc.rust-lang.org/std/process/struct.Command.html#method.args)!